### PR TITLE
feat: [CDS-36967]: Deployment type handled for service dashboard

### PR DESCRIPTION
**Pull request by bhavya-sinha**
##### Screenshots:
Screenshot 2022-05-02 at 2 49 21 PM
Screenshot 2022-05-02 at 2 57 06 PM
Screenshot 2022-05-02 at 2 57 31 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by bhavya-sinha** [comment]
retrigger titlecheck

**Review by kapilkumawat86 — DISMISSED** [review]

File: src/modules/70-pipeline/utils/DeploymentTypeUtils.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  NativeHelm: 'pipeline.nativeHelm',
+  Ssh: 'SSH',
+  ServerlessAwsLambda: 'pipeline.serviceDeploymentTypes.serverlessAwsLambda',
+  WinRm: 'pipeline.serviceDeploymentTypes.winrm'
+}
+
+export const deploymentTypeIcon: Record<string, IconName> = {
+  Kubernetes: 'service-kubernetes',
+  NativeHelm: 'service-helm',
+  ServerlessAwsLambda: 'app-aws-lambda',
```
Comment (chetanvekariya-harness) [inline src/modules/70-pipeline/utils/DeploymentTypeUtils.ts:23]: If you make any more changes to this PR then please include a change to use icon named **service-serverless-aws** for **ServerlessAwsLambda**

**Review by chetanvekariya-harness — APPROVED** [review]

File: src/modules/70-pipeline/utils/DeploymentTypeUtils.ts (hunk 1 above)
Comment (bhavya-sinha) [reply src/modules/70-pipeline/utils/DeploymentTypeUtils.ts:23]: Is this icon added recently? I found 'app-aws-lambda' in selectdeployment type

Comment (bhavya-sinha) [reply src/modules/70-pipeline/utils/DeploymentTypeUtils.ts]: Fixed

**Review by kapilkumawat86 — APPROVED** [review]

**Review by YounixIO — APPROVED** [review]

**Review by chetanvekariya-harness — APPROVED** [review]

**Comment by bhavya-sinha** [comment]
trigger cypress

**Pull request merged**